tests/acceptance/test_method_override.py

```diff
@@ -1,61 +0,0 @@
-"""
```

Comment:
This module needs to be added back into the acceptance tests. For that matter, we should be unit-testing the method override middleware quite rigorously as well.